.github/workflows/release.yaml

```diff
@@ -0,0 +1,12 @@
+name: Release
+
+on:
+    workflow_dispatch:
+    release:
+        types: [published]
+
```

Comment:
So this will only push once v0.10 is release?

Reply:
`.github/workflows/docker.yaml` will push after afery PR merged
`.github/workflows/release.yaml` will push on every release with a tag